content/en/news/releases/1.12.x/announcing-1.12/change-notes/index.md

```diff
@@ -0,0 +1,165 @@
+  ([Issue #35068](https://github.com/istio/istio/issues/35068))
+
+## istioctl
+
+- **Improved** Analyzers reports output to match the naming scheme expected by the API, i.e `<ns>/<name>` instead of `<name>.<ns>`.
+  ([Issue #35405](https://github.com/istio/istio/issues/35405))
+
+- **Improved** destination rule ca analyzer to show exact error line when using `istioctl analyze`,
+otherwise it will show the first line of its yaml configuration chunk.
+  ([Issue #22872](https://github.com/istio/istio/issues/22872))
+
+- **Updated** `istioctl x create-remote-secret` and `istioctl x remote-clusters` to the top level command, out of
+experimental.
+  ([Issue #33799](https://github.com/istio/istio/issues/33799))
+
+- **Added** `istioctl install` will now do `IST0139` analysis on webhooks.
+  ([Issue #33537](https://github.com/istio/istio/issues/33537))
+
+- **Added** `istioctl x remote-clusters` to list the remote clusters each `istiod` instance has API Server credentials for,
+and the service registry sync status of each cluster.
+  ([Issue #33799](https://github.com/istio/istio/issues/33799))
+
+- **Added** the pod alias `po` for users to use `istioctl x describe po`, which is consistent with `kubectl` command.
+  ([Pull Request #34802](https://github.com/istio/istio/pull/34802))
+
+- **Added** precheck now detects usage of Alpha Annotations.
+  ([Pull Request #35483](https://github.com/istio/istio/pull/35483))
+
+- **Added** `istioctl operator dump` now supports the `watchedNamespaces` argument to specify the namespaces the operator controller watches.
+  ([Issue #35485](https://github.com/istio/istio/issues/35485))
+
+- **Fixed** `istioctl operator` subcommands now support remote URLs specified in the `--manifests` argument.
+  ([Issue #34896](https://github.com/istio/istio/issues/34896))
+
+- **Fixed** `istioctl admin log` format.
+  ([Issue #34982](https://github.com/istio/istio/issues/34982))
+
+- **Fixed** APP pods (such as httpbin) can not be created if not using 'istio-system' as the Istio namespace to install Istio at the first time. And `istioctl install`, `istioctl tag set` and `istioctl tag generate` will be influenced. For example, user can set a specified namespace (`mesh-1` as an example) to install Istio via `istioctl install --set profile=demo --set values.global.istioNamespace=mesh-1 -y`
+  ([Issue #35539](https://github.com/istio/istio/issues/35539))
+
+- **Fixed** `istioctl bug-report` has the extra default system namespaces displayed when `--exclude` is not set.
+  ([Issue #35593](https://github.com/istio/istio/issues/35593))
+
+- **Fixed** the release tar URL by adding the patch version.
+  ([Pull Request #35712](https://github.com/istio/istio/pull/35712))
+
+- **Fixed** an issue in istioctl bug-report where --context and --kubeconfig were not being honored.
+  ([Issue #35574](https://github.com/istio/istio/issues/35574))
+
+## Documentation changes
```

Comment:
Nothing under this header, so can it be removed. Or we should see if there are any changes we want in this space. @craigbox anything to call out?